packages/compass-e2e-tests/helpers/commands/shell-eval.js

```diff
@@ -14,18 +15,22 @@ module.exports = function (app) {
 
   return async function (str, parse = false) {
     const { client } = app;
-    const shellContentElement = await client.$(Selectors.ShellContent);
-    if (!(await shellContentElement.isDisplayed())) {
-      await client.clickVisible(Selectors.ShellExpandButton);
-    }
+    let numLines;
 
-    const numLines = (await getOutputText()).length;
+    await retryWithBackoff(async function () {
+      const shellContentElement = await client.$(Selectors.ShellContent);
+      if (!(await shellContentElement.isDisplayed())) {
+        await client.clickVisible(Selectors.ShellExpandButton);
+      }
 
-    await client.clickVisible(Selectors.ShellInput);
-    // Might be marked with a deprecation warning, but can be used
-    // https://github.com/webdriverio/webdriverio/issues/2076
+      numLines = (await getOutputText()).length;
+
+      await client.clickVisible(Selectors.ShellInput);
+    });
```

Comment:
I can't find the root cause as the issue doesn't reproduce locally for me at all. This seems to work around it which I think is good enough for now at least